docs/overview/02-runtimes.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Runtimes
```

Comment:
What do you think about  combining this one with the one about source code. Runtimes doc is super short. Would look funky once we display only one doc at a time.
I.e "From code to function" : "pick a runtime and give us the code".  We can say which target runtimes are supported, and how you define code for the function and emphasise again that it is all users have to  do - kyma take cares about creating the workload

Reply:
I will try to combine those :)